bin/MadGraph5_aMCatNLO/gridpack_generation.sh

```diff
@@ -206,7 +206,7 @@ make_gridpack () {
         echo "copying bias module folder. Current dir:"
         pwd
         ls -lrth
-        cp -r $CARDSDIR/BIAS/* $MGBASEDIRORIG/Template/LO/Source/BIAS
+        cp -r $CARDSDIR/BIAS/* Template/LO/Source/BIAS
```

Comment:
Please do not change the general script. You can put a copy into your directory.

Reply:
@bbilin Let me know how to proceed, whether to revert back this script or not

Reply:
Thanks, but I do not understand why it fails with $MGBASEDIRORIG which is the directory  set as $(echo ${MG%$MG_EXT} | tr "." "_"). In line 190 of the current script you cd to this, and it is used everywhere. Is your Template directory under the MG folder? (I assume so)

 https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/gridpack_generation.sh#L190

Reply:
OK I get it now. thanks.